src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesDomainEventListener.kt

```diff
@@ -24,7 +24,7 @@ class ActivitiesDomainEventListener(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  @JmsListener(destination = "activity", containerFactory = "hmppsQueueContainerFactoryProxy")
+  @SqsListener("activity", factory = "hmppsQueueContainerFactoryProxy", maxMessagesPerPoll = "1", maxInflightMessagesPerQueue = "1")
```

Comment:
This makes me nervous after the experience with offender-events-ui. It took a few days for the RejectedTaskExecution error to happen in that project... @andymarke Do you think we need to process 1 message at a time?